src/target/devof12e/lcd.c

```diff
@@ -28,11 +37,25 @@ void LCD_Clear(unsigned int color) {
     (void)color;
     //printf("Clearing display\n");
     TW8816_ClearDisplay();
+#if SUPPORT_MULTI_LANGUAGE
+    memset(loadedchars, 0, sizeof(loadedchars));
+#endif
 }
 
 void LCD_PrintCharXY(unsigned int x, unsigned int y, u32 c)
 {
     c = TW8816_map_char(c);
+#if SUPPORT_MULTI_LANGUAGE
+    if (c > 0x300) {
+
+        if (height == 0) {
+            open_font("12normal");
```

Comment:
Can you find an alternative to hard-coding the font name in the code?

Reply:
currently, we handle the font name in LCD_String. the lcd_string under Text doesn't have such logic. We either hardcode it in lcd_string or here. Let me think a bit more and maybe I can come up some better approach.